pyrobosim/pyrobosim/data/test_world.yaml

```diff
@@ -40,13 +40,17 @@ robots:
 # PLANNING: Global planner
 global_path_planner:
   # Search graph planner from manually specified nodes
-  type: search_graph
-  max_edge_dist: 3.0
-  collision_check_dist: 0.05
+  # type: search_graph
+  # max_edge_dist: 3.0
+  # collision_check_dist: 0.05
   # Probabilistic Roadmap (PRM) planner
   # type: prm
   # max_connection_dist: 1.5
   # max_nodes: 100
+  # A* grid planner
+  type: astar_grid
+  resolution: 0.05
+  inflation_radius: 0.1
```

Comment:
Please revert these changes as they are used by all the other examples.

You can follow the RRT/PRM examples and do the same with setting the robot's path planner to `AStarGridPlanner` explicitly in your new demo file.